src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -28,10 +28,10 @@ object LocalBackend {
 class LocalBackend extends Backend {
 
   /**
-   * Executes the specified command line, using the supplied call and symbol store for expression evaluation.
+   * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
-  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], symbolStore: SymbolStore): Map[String, Try[WdlValue]] = {
+  override def executeCommand(commandLine: String, call: Call, taskOutputs: Seq[TaskOutput], scopedLookupFunction: ScopedLookupFunction): Map[String, Try[WdlValue]] = {
```

Comment:
1) I thought was easier to do this way, also it's not clear why it would be easier to log standard out with `-c`?
2) is definitely an issue and should be ticketed.